packages/main/src/request/index.ts

```diff
@@ -72,6 +72,20 @@ export const { actions, perform } = combineHandlers({
     })
   },
 
+  append(
+    connection: Connection,
+    flags: string[],
+    msgData: Buffer,
+    box: BoxSpecifier
+  ): R<string> {
```

Comment:
The result is a uid right? Instead of `string`, please use the type `imap.UID`. It is just an alias for `string`, but it helps anyone reading the code to understand what is represented by the result string.